packages/core/src/shared/utilities/workspaceUtils.ts

```diff
@@ -552,7 +552,7 @@ export async function collectFilesForIndex(
     }
 
     const isBuildOrBin = (filePath: string) => {
-        const k = /[/\\](bin|build|node_modules|env|\.idea)[/\\]/i
+        const k = /[/\\](bin|build|node_modules|env|\.idea|\.venv|venv)[/\\]/i
```

Comment:
do we need to worry about cases where the user has a non-conventional virtualenv path?